x-pack/plugins/security_solution/server/lib/telemetry/index.ts

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+import { UsageCollectionSetup } from 'src/plugins/usage_collection/server';
+import { ISavedObjectsRepository } from '../../../../../../src/core/server';
+import {
+  EndpointMetadataTelemetry,
+  getDefaultEndpointMetadataTelemetry,
+  getEndpointMetadataTelemetryFromFleet,
+} from './endpoint_metadata_telemetry';
+
+const SIEM_USAGE_COLLECTOR_TYPE = 'security_solution';
+export async function createSiemTelemetry(
+  usageCollector: UsageCollectionSetup,
+  savedObjectsClient: ISavedObjectsRepository
+) {
+  const collector = usageCollector.makeUsageCollector({
+    type: SIEM_USAGE_COLLECTOR_TYPE,
+    isReady: () => true,
```

Comment:
Does this need to wait on the ingest manager being initialized? I suppose if no saved objects existed then we jus wouldn't find them right?

Reply:
Yea and we wouldn't send empty telemetry until the next time this runs. @TinaHeiligers is there a better way to implement this?